.vscode/settings.json

```diff
@@ -139,5 +138,8 @@
         "--log-level=DEBUG",
         "."
     ],
-    "python.testing.unittestEnabled": false
+    "python.testing.unittestEnabled": false,
+    "cSpell.words": [
+        "SOBOL"
+    ]
```

Comment:
These go in different .cspell file, please move them there and revert the rest of the changes in this file.